project/test/resources/OPConsultantRecord-Output.json

```diff
@@ -1,2688 +1,2143 @@
+        "profile": [ "https://nrces.in/ndhm/fhir/r4/StructureDefinition/Observation" ]
+      },
+      "status": "final",
+      "category": [ {
+        "coding": [ {
+          "system": "http://terminology.hl7.org/CodeSystem/observation-category",
+          "code": "vital-signs",
+          "display": "Vital Signs"
+        } ],
+        "text": "Vital Signs"
+      } ],
+      "code": {
+        "coding": [ {
+          "system": "http://loinc.org",
+          "code": "39156-5",
+          "display": "Body mass index (BMI) [Ratio]"
+        } ],
+        "text": "Body Mass Index"
+      },
+      "subject": {
+        "reference": "urn:uuid:9886e72f-be96-48f1-8c12-c98dc1d79033",
+        "display": "Patient"
+      },
+      "effectiveDateTime": "2024-07-01T16:29:32+05:30",
+      "valueQuantity": {
+        "value": 62.0,
+        "unit": "kg/m2",
+        "system": "http://unitsofmeasure.org",
+        "code": "kg/m2"
+      }
+    }
+  }, {
+    "fullUrl": "urn:uuid:9d7e0de1-e040-42ea-a246-76aa6d85e34b",
+    "resource": {
+      "resourceType": "Observation",
+      "id": "9d7e0de1-e040-42ea-a246-76aa6d85e34b",
+      "meta": {
+        "versionId": "0",
+        "lastUpdated": "2024-07-01T16:29:32.573+05:30",
+        "profile": [ "https://nrces.in/ndhm/fhir/r4/StructureDefinition/Observation" ]
+      },
+      "status": "final",
+      "category": [ {
+        "coding": [ {
+          "system": "http://terminology.hl7.org/CodeSystem/observation-category",
+          "code": "vital-signs",
+          "display": "Vital Signs"
+        } ],
+        "text": "Vital Signs"
+      } ],
+      "code": {
+        "coding": [ {
+          "system": "http://loinc.org",
+          "code": "882-1",
+          "display": "ABO and Rh group [Type] in Blood"
+        } ],
+        "text": "Blood Group"
+      },
+      "subject": {
+        "reference": "urn:uuid:9886e72f-be96-48f1-8c12-c98dc1d79033",
+        "display": "Patient"
+      },
+      "valueCodeableConcept": {
+        "coding": [ {
+          "system": "http://snomed.info/sct",
+          "code": "365637002",
+          "display": "Finding of ABO blood group"
+        } ],
+        "text": "A+"
+      }
+    }
+  }, {
+    "fullUrl": "urn:uuid:9886e72f-be96-48f1-8c12-c98dc1d79033",
+    "resource": {
+      "resourceType": "Patient",
+      "id": "9886e72f-be96-48f1-8c12-c98dc1d79033",
+      "meta": {
+        "versionId": "0",
+        "lastUpdated": "2024-07-01T16:29:32.574+05:30",
+        "profile": [ "https://nrces.in/ndhm/fhir/r4/StructureDefinition/Patient" ]
+      },
+      "identifier": [ {
+        "id": "123",
+        "type": {
+          "coding": [ {
+            "system": "http://terminology.hl7.org/CodeSystem/v2-0203",
+            "code": "MR",
+            "display": "Medical record number"
+          } ],
+          "text": "Medical record number"
+        },
+        "system": "urn:health:information:provider:system",
+        "value": "NCG"
```

Comment:
please provide representative data - NCG is not a Medical Record number for patient .. you may provide representative data like say "UHID-109123" - typically this would be the patient health id thats particular to a provider.